bin/package_build.py

```diff
@@ -387,7 +382,6 @@ def pds(q):
 	else:
 		print(
 			"Usage:\n./package_build <exact_file_name.json>\n\t\t\t[if data is from PDS]"
-			"\n./package_build.py debian\n\t\t\t[if data is from Debian]"
```

Comment:
Oops! I missed that this was still removed in your follow-up PR. Can you submit another PR to add it back?